### PR TITLE
Add option to include prefix in the exported variables

### PR DESCRIPTION
**Pull request by jkrajniak**

### CODE REVIEW

File: cmd/assume-role-arn/main.go [hunk 1]
```diff
@@ -201,27 +203,34 @@ func isCredentialsValid(sess *session.Session) bool {
...
+	os.Unsetenv(prefixEnvVar("AWS_ROLE_NAME"))
+	os.Unsetenv(prefixEnvVar("AWS_ACCOUNT_ID"))
+	os.Unsetenv(prefixEnvVar("AWS_PROFILE_NAME"))
+}
+
+func prefixEnvVar(envVarName string) string {
+	if len(envPrefix) == 0 {
+		return envVarName
+	}
+	return fmt.Sprintf("%s_%s", envPrefix, envVarName)
```
Comment (gzbd) [inline cmd/assume-role-arn/main.go:233]: IMO, it should just concatenate the prefix with the variable name (no underscore).

File: README.md [hunk 2]
```diff
@@ -34,6 +34,7 @@ Available flags:
 *  `-profile profile_name` - the name of AWS profile (from $HOME/.aws/config)
 *  `-ignoreCache` - ignore the credentials stored in the cache
 *  `-skipCache` - skip the credential cache
+*  `-envprefix` - add prefix for the exported ENVs (default: "")
```
Comment (tdi) [inline README.md:37]:
```suggestion
*  `-envprefix` - add prefix to exported environmental variables (default: "")
```

File: cmd/assume-role-arn/main.go (hunk 1 above)
Comment (tdi) [inline cmd/assume-role-arn/main.go:233]:
```suggestion
	return fmt.Sprintf("%s%s", envPrefix, envVarName)
```

Suggesting along @gzbd suggestion

**Comment by jkrajniak** [comment]
> What's the use case here? Just curious

Yeah, use case. I have a script that takes these variables in the command line as the input parameters. (e.g. `./script -a access_key_id -s secret_key -t session_token`). Obviously I could run it with the existing version by
```bash
$ $(assume-role-arn-linux)
$ ./script -a $AWS_ACCESS_KEY_ID ...
```
but I do not want to have these default variables set up in the shell. Hence, by using the prefix I'm safe and can run it in this way
```bash
$ $(assume-role-arn-linux -envprefix pref_)
$ ./script -a $pref_AWS_ACCESS_KEY_ID ...
```

**Pull request merged**